manager-bundle/tests/Command/ContaoSetupCommandTest.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerBundle\Tests\Command;
+
+use Contao\ManagerBundle\Command\ContaoSetupCommand;
+use Contao\ManagerBundle\Process\ProcessFactory;
```

Comment:
It seems that this class no longer exists?

Reply:
Removed in 845b0a057dc05e896411708d76984b8f3b76df2e